app/commands/config.go

```diff
@@ -301,7 +302,7 @@ func setupLingo(c *cli.Context) (string, error) {
 	// TODO (Junyu) set proper Perforce Username and password
```

Comment:
Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.